### PR TITLE
disable map scroll zoom 

### PR DESCRIPTION
**Pull request by jackcarlisle**
## ready for review

* disables scrollwheel zoom on the google map component  
#530

### CODE REVIEW

**Comment by jruts** [comment]
What a huge commit :fearful:

File: src/components/destination/index.js [hunk 1]
```diff
@@ -41,6 +41,7 @@ class DestinationFullPage extends Component {
                 ref={(map) => (map) => console.log(map)}
```
Comment (lennym) [inline src/components/destination/index.js]: Unrelated to this PR, but worth noting... Do we want this in prod?

**Comment by nelsonic** [comment]
@jackcarlisle (I know it's a pain but) please raise an issue in JIRA before diving into fixing things so that we can show output to the decision makers... thanks! ❤️

File: src/components/destination/index.js (hunk 1 above)
Comment (nelsonic) [inline src/components/destination/index.js]: Probably not...

**Comment by jackcarlisle** [comment]
@nelsonic sorry! been in the habit of using github issues, will raise one now

**Pull request merged**